resources/sample-jwks/authorization-example-jwks-and-signatures.md

```diff
@@ -0,0 +1,52 @@
+```
+#!/usr/bin/env python
```

Comment:
We said we'd include a rendered ipython notebook and a rew ipython ntoebook. Can you include the outputs here so a user looking at this rendering sees the whole story? (Also, did you create this .md file by hand? Would be good to use one of the built-in jupyter output formats to make this easier to update/maintain over time.)

Reply:
@ricky1uphealth make sure the display output
display the command while using the CLI to get the output

Reply:
fixed here https://github.com/HL7/bulk-data/pull/10/commits/ad5cba4c18ff42427e5d3116f221deb7ef2dc044